.env.example

```
@@ -0,0 +1,8 @@
+DEBUG=1
+APPLY_MIGRATIONS=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPLY_MIGRATIONS key should go before the DEBUG key

(UnorderedKey)